packages/react/src/components/ImageUploader/index.js

```diff
@@ -0,0 +1,90 @@
+import React from "react";
+import PropTypes from "prop-types";
+import cx from "classnames";
+
+import FileUploader from "../FileUploader";
+import IconButton from "../IconButton";
+
+import { ICONS } from "../../constants";
+
+const toBase64 = file => {
+  const reader = new FileReader();
+
+  return new Promise(resolve => {
+    reader.onload = ({ target }) => resolve(target.result);
+    reader.readAsDataURL(file);
+  });
+};
+
+const ImageUploader = ({ onChange, ...props }) => {
+  const [file, setFile] = React.useState(undefined);
```

Comment:
You prefer to do this like that as opposed to `import React, { useState } from 'react'`?

Reply:
Yep 🙂

Reply:
👍